src/main/resources/avro/bdg.avdl

```diff
@@ -410,6 +412,19 @@ record Variant {
    is shared across all alleles in the same VCF record.
    */
   union { boolean, null } somatic = false;
+
+  /**
+  True if this site/block represents a gVCF line where no alternate allele was
+  called. This includes both gVCF modes (ERC and BP_RESOLUTION).
+   */
+  union { boolean, null } isGvcfReferenceModel = false;
```

Comment:
Ah yes, I'll revise this. "gVCF reference model" = a site where there is no confident alt, and where all information represents the probability of a ref call vs. "all non ref alleles".